### DISCUSSION TITLE
利用备份的数据库恢复之后提示：更新公共模板失败, 请稍后刷新重试

### DISCUSSION THREAD

**Discussion by lxt20180608** [qd-today/qd#514] [Q&A]
错误代码：undefined,
错误原因：undefined
错误代码：1006,
错误原因：

更新公共模板失败, 请稍后刷新重试, 请勿连续刷新！

备份恢复后已重启QD容器

**Comment by a76yyyy** [comment]
看后台日志显示什么错误
